Tasks/SonarQubePostTest/task.json

```diff
@@ -12,7 +12,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 47
+    "Patch": 48
```

Comment:
Bot pre and post task use the Common file with the HTTPS fix, so I am incrementing them both.